src/walletSdk/Auth/index.ts

```diff
@@ -52,13 +56,31 @@ export class Auth {
     }
   }
 
-  // TODO - add signing with client account functionality
-  sign(accountKp: Keypair, challengeTx, network) {
-    const transaction = StellarSdk.TransactionBuilder.fromXDR(
-      challengeTx,
-      network
+  sign(
```

Comment:
Both this and challenge functions should be private

Reply:
👍 https://github.com/stellar/typescript-wallet-sdk/pull/13/commits/27ec5a5630f9b175df0d6c59d75a24b8eef9cbb4